lang/cs/Org.Apache.REEF.Examples/Tasks/HelloTask/HelloTask.cs

```diff
@@ -66,7 +66,7 @@ public byte[] Call(byte[] memento)
                 Console.WriteLine("IP Address: {0}", _nameClient.Lookup("abc"));
             }
             PrintGuestList();
-            Thread.Sleep(5 * 1000);
+            Thread.Sleep(1000);
```

Comment:
It was needed at one of previous iterations, but now tests pass both locally and [on AppVeyor](https://ci.appveyor.com/project/tcNickolas/reef/build/134-REEF-1388-t) without this, so probably not needed any more